web/examples/craft/main.dart

```diff
@@ -93,3 +99,22 @@ void startCraft() {
     print("$fps fps, "+world.debugString());
   });
 }
+
+/// Returns the seed provided by the URL's query parameters, or null if no seed is found.
+int _getSeed() {
+  String seedQueryParam = Uri.base.queryParameters["seed"];
+  if (seedQueryParam == null) {
+    return null;
```

Comment:
Since `nextInt` can only be a natural number (0 or more) and I try to keep my types more strict (even though JS is playdough and allows such a thing). So instead of `null` please use `-1` and change the check to `if (seed < 0)`.